lib/backend/k8s/k8s.go

```diff
@@ -258,6 +275,8 @@ func (c *KubeClient) Create(d *model.KVPair) (*model.KVPair, error) {
 	switch d.Key.(type) {
 	case model.GlobalConfigKey:
 		return c.createGlobalConfig(d)
+	case model.IPPoolKey:
```

Comment:
There isn't much cost, but I don't think it matters much either way.  I like the way I've got it more though :)